test/Range.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { Range } from '../src/Range';
+
+describe('Range', () => {
+    let range: Range;
+    beforeAll(() => {
+        range = new Range([0, 70], [[2, 2], [4, 5], [10, 15], [32, 40], [50, 50]]).invert().invert();
```

Comment:
I don't think you should use `invert` here. If that method has a bug then other tests will fail. That can be very confusing to debug.